taxcalc/params.json

```diff
@@ -1265,5 +1265,29 @@
         "value": [  [400000, 450000, 225000, 425000, 450000, 225000],
                     [406750, 457600, 228800, 432200, 457600, 228800],
                     [413200, 464850, 223425, 439000, 464850, 223425]]
+    },
```

Comment:
We should have some param validation that guarantees the user doesn't change both of these at the same time.